src/Totals.js

```diff
@@ -43,7 +43,9 @@ class Totals extends Component {
         </section>
         <section>
           <h3>Potential Profits</h3>
-          <h2>${(this.totalProfit().toFixed(2)) - this.totalCost()}</h2>
+
```

Comment:
I'm a bit thrown off seeing the `h3` in your code before the `h2`... when thinking about these headings semantically, `h3` is a sub-element of `h2` and  one `section` element should have 1 heading.